theme/static/js/hs_file_browser.js

```diff
@@ -413,39 +413,42 @@ function bindFileBrowserItemEvents() {
     });
 }
 
-function showFileTypeMetadata() {
-    var logical_file_id = $("#fb-files-container li.ui-selected").attr("data-logical-file-id");
-    var logical_type = $("#fb-files-container li").children('span.fb-logical-file-type').attr("data-logical-file-type");
-    var resource_mode = $("#resource-mode").val();
-    resource_mode = resource_mode.toLowerCase();
-    var url = "/hsapi/_internal/" + logical_type + "/" + logical_file_id + "/" + resource_mode + "/get-file-metadata/";
-    $(".file-browser-container, #fb-files-container").css("cursor", "progress");
-    var calls = [];
-    calls.push(get_file_type_metadata_ajax_submit(url));
-    // Wait for the asynchronous calls to finish to get new folder structure
-    $.when.apply($, calls).done(function (result) {
-        var json_response = JSON.parse(result);
-        $("#fileTypeMetaDataTab").html(json_response.metadata);
-        $(".file-browser-container, #fb-files-container").css("cursor", "auto");
-        showMetadataFormSaveChangesButton();
-        initializeDatePickers();
-        setFileTypeSpatialCoverageFormFields();
-        var $spatial_type_radio_button_1 = $("#div_id_type_filetype").find("#id_type_1");
-        var $spatial_type_radio_button_2 = $("#div_id_type_filetype").find("#id_type_2");
-        if (logical_type === "GeoRasterLogicalFile") {
-            $spatial_type_radio_button_1.prop("checked", true);
-            $("#div_id_type_filetype input:radio").trigger("change");
-            $spatial_type_radio_button_1.attr('onclick', 'return false');
-            $spatial_type_radio_button_2.attr('onclick', 'return false');
-        }
-        else {
-            if ($spatial_type_radio_button_1.attr('checked') == 'checked') {
-                $spatial_type_radio_button_1.prop("checked", true);
-            }
-            else {
-                $spatial_type_radio_button_2.prop("checked", true);
-            }
-        }
+function showFileTypeMetadata(){
+     var logical_file_id = $("#fb-files-container li.ui-selected").attr("data-logical-file-id");
+     if (logical_file_id.length == 0){
```

Comment:
If the user selects multiple files it seems it will get the logical_file_id of the last selected file. Then the metadata displayed will be for that file. But there is no way for the user to know the displayed file metadata is for which file. It would be better not to call the showFileTypeMetadata() if more than one file is selected. But again that causes another problem where multiple files under a folder have the same logical file id. That is the case for the raster file type. In that case it really does not matter if the user selects one or more files. The displayed metadata is same for each of those files. At least this PR will fix the JS error for now. Will do further improvement to this logic if users find issues with the current implementation.